sdk/storage/azblob/appendblob/models.go

```diff
@@ -73,8 +74,12 @@ func (o *CreateOptions) format() (*generated.AppendBlobClientCreateOptions, *gen
 
 // AppendBlockOptions contains the optional parameters for the Client.AppendBlock method.
 type AppendBlockOptions struct {
-	// Specify the transactional crc64 for the body, to be validated by the service.
-	TransactionalContentCRC64 []byte
+	// Let the SDK hash for you (providing the hash type(s) specified voids their relevance in this option).
+	// nil = None default
+	TransactionalValidationOption exported.TransferValidationType
+
+	// Specify the transactional crc64 for the body, to be validated by the service. Should be hashed using hashing.CRC64Table or hashing.CRC64Polynomial
+	TransactionalContentCRC64 uint64
```

Comment:
If the SDK performs the hashing based on the value of the above field, does this field still need to exist?

Reply:
These are for the user supplying their own hash. On a technicality, they could still be used to validate hashing functionality in testing, or a user could have their own pipeline where they've already hashed the incoming data and don't want to re-hash it.

Reply:
Does this mean that the two options are mutually exclusive?

Reply:
They are.